src/features/user/Profile/styles/RedeemModal.module.scss

```diff
@@ -84,6 +84,12 @@
   min-height: 275px;
   min-width: 438px;
   padding: 20px 56px;
+  >div:nth-child(5) {
+    padding-top: 15px;
+  }
```

Comment:
What is this style for? If this is for the padding above the button, it will cause issues - what happens if I change the order of the div elements, or I add another div element above the button.

Please change this.

Reply:
@sunilsabatp This is done. One minor change -> I noticed that

.redeemAnotherCodeDiv and .redeemCodeButtonContainer have different styles. Should these be the same (for consistency in spacing)? Also, should one class satisfy the purpose?

Reply:
Styles applied were not reflected due to nesting / application of incorrect class. Corrected here - cc264bd68674b084d92c95552b95f7af6adda7a4